packages/core/src/app/order/OrderSummaryPrice.tsx

```diff
@@ -41,109 +42,102 @@ function isNumberValue(displayValue: number | ReactNode): displayValue is number
+                        className,
+                    )}
                 >
-                    <div
-                        aria-live="polite"
-                        className={classNames(
-                            'cart-priceItem',
-                            'optimizedCheckout-contentPrimary',
-                            className,
-                        )}
-                    >
-                        <span className="cart-priceItem-label">
-                            <span data-test="cart-price-label">
-                                {label}
-                                {'  '}
-                            </span>
-                            {currencyCode && (
-                                <span className="cart-priceItem-currencyCode">
-                                    {`(${currencyCode}) `}
-                                </span>
-                            )}
-                            {onActionTriggered && actionLabel && (
-                                <span className="cart-priceItem-link">
-                                    <a
-                                        data-test="cart-price-callback"
-                                        href="#"
-                                        onClick={preventDefault(onActionTriggered)}
-                                    >
-                                        {actionLabel}
-                                    </a>
-                                </span>
-                            )}
+                    <span className="cart-priceItem-label">
+                        <span data-test="cart-price-label">
+                            {label}
+                            {'  '}
                         </span>
-
-                        <span className="cart-priceItem-value">
-                            <span data-test="cart-price-value">
-                                {isNumberValue(displayValue) ? (
-                                    <ShopperCurrency amount={displayValue} />
-                                ) : (
-                                    displayValue
-                                )}
+                        {currencyCode && (
+                            <span className="cart-priceItem-currencyCode">
+                                {`(${currencyCode}) `}
                             </span>
-
-                            {superscript && (
-                                <sup data-test="cart-price-value-superscript">{superscript}</sup>
+                        )}
+                        {onActionTriggered && actionLabel && (
+                            <span className="cart-priceItem-link">
+                                <button
+                                    data-test="cart-price-callback"
+                                    disabled={isDisabled}
+                                    onClick={preventDefault(onActionTriggered)}
+                                    style={{color: isDisabled ? '#ddd' : '#4496f6'}}
```

Comment:
To get the colour from the theme, you might want to use a class name from the theme's CSS file.
I think the related class names are either:
```
.optimizedCheckout-buttonPrimary[disabled],.optimizedCheckout-buttonPrimary[disabled].active,.optimizedCheckout-buttonPrimary[disabled].is-active,.optimizedCheckout-buttonPrimary[disabled].is-loading,.optimizedCheckout-buttonPrimary[disabled]:focus,.optimizedCheckout-buttonPrimary[disabled]:hover
```
or
```
.optimizedCheckout-buttonSecondary[disabled],.optimizedCheckout-buttonSecondary[disabled].active,.optimizedCheckout-buttonSecondary[disabled].is-active,.optimizedCheckout-buttonSecondary[disabled].is-loading,.optimizedCheckout-buttonSecondary[disabled]:focus,.optimizedCheckout-buttonSecondary[disabled]:hover
```
All class names can be found at [here](https://cdn11.bigcommerce.com/s-pyfdp1ymny/stencil/55c3c420-d9d0-013a-571a-12ed63bdf519/css/optimized-checkout-5fef6070-0fdd-013b-da77-062f8167c164.css).

Reply:
Yes, I saw this classes.
But the problem is that for now we display this button like a link `<a href=... />` and `optimizedCheckout-button...` classes is looks like a button. So by adding this classes we will change appearance what is not good.
But for link (`<a />`) we don't have a disable state and styles for this state.

So, @animesh1987 , maybe we can leave it as it is with a `<a href=... />` and hide this button in case user can't click on it?

Examples with current implementation and button classes
<img width="2560" alt="Screenshot 2023-12-18 at 12 51 02" src="https://github.com/bigcommerce/checkout-js/assets/9430298/f6b3a5bd-87ef-49d8-85e9-73ca964ac01d">

Reply:
Feel free to add disabled styles for `<a />` tag.

Reply:
Hi @animesh1987 @bc-peng 
I have added styles for disabled link(video updated)
can you please check?